inc/sainexthopgroup.h

```diff
@@ -219,6 +255,30 @@ typedef enum _sai_next_hop_group_member_attr_t
      */
     SAI_NEXT_HOP_GROUP_MEMBER_ATTR_MONITORED_OBJECT,
 
+    /**
+     * @brief Object index in the fine grain ECMP table.
+     *
+     * Index specifying the member's order.
+     * Should only be used if the type of owning group is SAI_NEXT_HOP_GROUP_TYPE_FINE_GRAIN_ECMP.
+     *
+     * @type sai_uint32_t
+     * @flags CREATE_ONLY
+     * @default internal
+     */
+    SAI_NEXT_HOP_GROUP_MEMBER_ATTR_INDEX,
```

Comment:
should we add min and max where min is 0 and max is SAI_NEXT_HOP_GROUP_ATTR_REAL_SIZE - 1

Reply:
Done